platform-dom/src/main/java/com/softicar/platform/dom/elements/testing/node/tester/AbstractDomNodeTester.java

```diff
@@ -231,11 +231,13 @@ public boolean containsNoText() {
 
 	// ------------------------------ retrieve textual content ------------------------------ //
 
+	// TODO this method should be renamed to getAllText (or similar)
```

Comment:
This method deserves some javadoc. Particularly, that should answer the question whether text in the node itself would be included in the result (as opposed to: only the texts in child nodes).

Reply:
Also, `PLAT-568` should be referenced.

Reply:
ok